pages/app/PartnerDetail.js

```diff
@@ -15,6 +27,11 @@ class PartnerDetail extends Page {
   */
   componentDidMount() {
     this.props.getPartnerData(this.props.url.query.id);
```

Comment:
I think we should use server rendering here => `this.props.getPartnerData(this.props.url.query.id)`